win32ss/user/ntuser/desktop.c

```diff
@@ -260,6 +261,16 @@ InitDesktopImpl(VOID)
                       SynchronizationEvent,
                       FALSE);
 
+    /* Check if user has disabled window snapping */
+    DWORD dwValue = 0;
+    BOOL ret = RegReadUserSetting(L"Control Panel\\Desktop",
+                                            L"WindowArrangementActive",
+                                            REG_SZ, &dwValue, sizeof(dwValue));
+    if(ret) {
+        dwValue = atoi((PCHAR)&dwValue);
+        g_AllowSnappingWindows = dwValue == 1;
```

Comment:
Only on 1 it should be active?
I would assume that windows uses something more like `g_AllowSnappingWindows = dwValue != 0`

Reply:
Just checked out of curiosity (win10 20h2), if the registry entry is a number then it does indeed work if `dwValue != 0`; otherwise it remains disabled. I'll change it to `g_AllowSnappingWindows = dwValue != 0`, can switch to `strtol` if you want me to match windows' behavior accurately